3rdparty/d3drm/CMakeLists.txt

```diff
@@ -2,7 +2,33 @@ project(wine_d3drm LANGUAGES C)
 
 set(CMAKE_C_CLANG_TIDY)
 
-add_library(d3drm-wine SHARED EXCLUDE_FROM_ALL
+if(MSVC)
+    # FIXME: this is wrong.
+    if(CMAKE_SIZEOF_VOID_P EQUAL 4)
+        set(def "${CMAKE_CURRENT_SOURCE_DIR}/d3dxof32.def")
+    else()
+        set(def "${CMAKE_CURRENT_SOURCE_DIR}/d3dxof64.def")
+    endif()
+    set(imp "${CMAKE_CURRENT_BINARY_DIR}/d3dxof.lib")
+    set(exp "${CMAKE_CURRENT_BINARY_DIR}/d3dxof.exp")
```

Comment:
I was able to get everything building and linking with MSVC targeting x64.
There is now an issue with d3drm's dependency on d3dxof: does a 64-bit d3dxof.dll even exist?

[DirectXFileCreate](https://learn.microsoft.com/en-us/windows/win32/direct3d9/directxfilecreate)

Perhaps we need to copy its code from wine as well.

Reply:
I think the (initial) easiest solution would be to load `DirectXFileCreate` dynamically (using `LoadLibrary` + `GetProcAddress`).
We might not even need `d3dxof.dll` library.

Reply:
Never mind, looks like it is used to deserialize data.

Reply:
I made this an option so you can try by configuring with `-DWINE_D3DRM_DYNAMIC_D3DXOF=ON/OFF`.